### PR TITLE
Task47

### PR DESCRIPTION
**Pull request by katka-juhasova**
- renamed curvePerpendicular.ipynb -> curve_perpendicular.ipynb
- added new notebook for data preprocessing for NN
- added new file containing input data for NN

### CODE REVIEW

**Comment by Kesleroman** [comment]
@katka-juhasova do you know why there are small differences in coordinates of new laps?

File: data/laps/lap1-0.csv [hunk 1]
```diff
@@ -1,2304 +1,2304 @@
...
-48.0492516,17.567878,6.07,220.74945000000002,48.0492479,17.5678731,221.10654000000002
-48.0492479,17.5678731,5.82,221.10654000000002,48.0492443,17.5678686,221.55495000000002
-48.0492443,17.5678686,5.3,221.55495000000002,48.049241099999996,17.5678646,220.3692
-48.049241099999996,17.5678646,5.0,220.3692,48.0492381,17.5678608,219.94348000000002
+LAT,LON,GSPEED,CRS,NLAT,NLON,NCRS,CMD
+48.0492306,17.567850699999997,0.0,0.0,48.049231299999995,17.567849399999997,0.0,11.0116384005144
+48.049231299999995,17.567849399999997,0.04,0.0,48.0492305,17.5678495,0.0,-12.070141112379401
+48.0492305,17.5678495,0.01,0.0,48.0492292,17.5678499,0.0,-19.6874865336079
+48.0492292,17.5678499,0.01,0.0,48.0492283,17.5678498,0.0,-13.9127673190133
+48.0492283,17.5678498,0.02,0.0,48.0492274,17.56785,0.0,-13.781592113715
```
Comment (Kesleroman) [inline data/laps/lap1-0.csv:6]: For example, here

**Comment by Kesleroman** [comment]
And I haven't understood why you cropped the first 1000 measurements of the lap1-0.
All the other seems to be fine.

File: data/laps/lap1-0.csv (hunk 1 above)
Comment (katka-juhasova) [reply data/laps/lap1-0.csv:6]: I really have no idea, I think I've already seen it happen in some cases but I don't know why

**Comment by katka-juhasova** [comment]
In the beginning of the lap1-0 there's a big mess where the vehicle is obviously not moving and the coordinates just create one messy spot

File: data/laps/lap1-0.csv (hunk 1 above)
Comment (Kesleroman) [reply data/laps/lap1-0.csv:6]: Ideally, we need to recreate the file differences.cvs.
However, it seems that differences are neglectable.

**Comment by Kesleroman** [comment]
I don't see the button Approve here.
You can merge if you are not going to change something else.

**Pull request merged**